packages/yoastseo/spec/specHelpers/scoring/seoAssessorTests.js

```diff
@@ -41,40 +41,41 @@ export function checkAssessmentAvailability( assessor, isProductAssessor = false
 		defaultAssessments.pop();
 	}
 
-	let requiresTextAssessments = [ "images", "externalLinks", "internalLinks" ];
+	let commonAssessments = [ "images", "externalLinks", "internalLinks" ];
 	if ( isCollection || isStoreBlog || isTaxonomy ) {
-		requiresTextAssessments = [];
+		commonAssessments = [];
 	}
 	if ( isProduct ) {
-		requiresTextAssessments = [ "images" ];
+		commonAssessments = [ "images" ];
 	}
 
 	const requiresKeyphraseAssessments = [ "introductionKeyword" ];
 	if ( isStoreBlog ) {
 		requiresKeyphraseAssessments.pop();
 	}
 
-	const mostAssessments = [ ...defaultAssessments, ...requiresTextAssessments, ...requiresKeyphraseAssessments ];
+	const textAssessments = [ ...defaultAssessments, ...commonAssessments ];
+	const mostAssessments = [ ...textAssessments, ...requiresKeyphraseAssessments ];
```

Comment:
Not sure if I understand the naming and categorization of assessment groups here 😅 
* what are `commonAssessments`? And why shouldn't they be grouped together with the `defaultAssessments`? From what I understand, the assessment grouping here was based on assessment applicability (previously: always applicable, requires text, requires keyphrase). Since `requiresTextAssessments` are now always applicable, shouldn't they be grouped together with the `defaultAssessments`?
* Why are `textAssessments` called this way?

Maybe some code comments would also help here.